projects/storefrontlib/cms-components/product/product-list/product-list.module.ts

```diff
@@ -38,7 +37,7 @@ import { ProductViewComponent } from './product-view/product-view.component';
   imports: [
     AtMessageModule,
     CommonModule,
-    FeaturesConfigModule,
+
```

Comment:
I am not sure if this is important, the removal of the `FeatureConfigModule` created an empty space.

Reply:
```suggestion
```